### PR TITLE
Clarify which XML attribute of a wheel slot is to be used when

### PR DESCRIPTION
**Pull request by petrvanekrobe**

### CODE REVIEW

File: gdtf-spec.md [hunk 1]
```diff
@@ -534,11 +536,25 @@ done via the wheel slot index. The wheel slot index of a slot is derived
...
+function of the linked channel set defines which XML attributes of a wheel slot
+should be utilized and in what order of importance:
+
+| GDTF Attribute Name | Utilized XML attribute                                      |
+|----|----|----|
+| Color(n)              | MediaFileName, Filter, Color                              |
+| Gobo(n)               | MediaFileName                                             |
+| Prism(n)              | Facet                                                     |
+| AnimationWheel(n)     | MediaFileName                                             |
+| AnimationSystem(n)    | AnimationSystem                                           |
```
Comment (danielmuellerma) [inline gdtf-spec.md:554]: together with MediaFileName

Comment (petrvanekrobe) [reply gdtf-spec.md]: Thank you, added

File: gdtf-spec.md [hunk 2]
```diff
@@ -534,11 +536,25 @@ done via the wheel slot index. The wheel slot index of a slot is derived
 from the order of a wheel's slots. The wheel slot index is normalized to
 1.
 
-If the wheel slot has a prism, it has to have one or several children
-called [prism facet](#user-content-prism-facet ). If the wheel slot has an
+If the wheel slot describes a prism, it has to have one or several children
+called [Prism Facet](#user-content-prism-facet ). If the wheel slot describes an
 AnimationWheel, it has to have one child called [Animation
```
Comment (danielmuellerma) [inline gdtf-spec.md:541]: I think this should be AnimationSystem

**Comment by petrvanekrobe** [comment]
- Change the SpecialFunction to something better
- Add optional to the special function 
- Rework the explanation sentence

**Comment by petrvanekrobe** [comment]
- Open an issue for the builder to do error detection based on the new table
- We need a new issue for the libMVRGDTF for the "SpecialFunction" attribute

File: gdtf-spec.md [hunk 3]
```diff
@@ -520,11 +520,13 @@ currently defined XML attributes of the wheel slot are specified in
 | Color              | [ColorCIE](#user-content-attrtype-colorcie )   | Color of the wheel slot, Default value: {0.3127, 0.3290, 100.0 } (white) For Y give relative value compared to overall output defined in property Luminous Flux of related Beam Geometry (transmissive case).                                                                                                                     |
 | Filter             | [Node](#user-content-attrtype-node )       | Optional. Link to filter in the physical description; Do not define color if filter is used; Starting point: Filter Collect                                                                                                                                                                                                       |
```
Comment (danielmuellerma) [inline gdtf-spec.md:521]: As we now have defined the order we can remove "Do not define color if filter is used"

**Comment by bbudzon** [comment]
I gave this another look and I think it looks good @petrvanekrobe!

**Comment by danielmuellerma** [comment]
this fixes #214

**Pull request merged**